deployments/cloudfoundry/tile/scripts/README.md

```diff
@@ -0,0 +1,12 @@
+# Tanzu Tile Scripts
```

Comment:
Is there any value in adding a github workflow for these scripts?

Reply:
There would be value when moving to automate the Tanzu tile test and release process. The `setup_tanzu.sh` script would be more useful for a workflow, and would be good to test on any change that could impact the Tanzu tile's functionality.

The `generate_osdf.py` script would only be useful to run during the release process for a new Tanzu tile.